### DISCUSSION TITLE
Export as "read-only recommended" document?

### DISCUSSION THREAD

**Discussion by SVR-VY** [dfinke/ImportExcel#1334] [Q&A]
Hi,

I needed a way to write an Excel-file from a device that does not have Excel installed. I found this powershell module and I have to say thanks, it saved my day. Kudo's on the work!

The file I write needs to have the option enabled where, when you open it, excel prompts to open the document read only. This is a very useful but well hidden feature. (In excel: file - save as - browse, in the 'Save as Dialog box': Tools - General options - "Read Only Recommended")

When you want to open the file it will ask you to open read only: [example screenshot here](https://i.imgur.com/5Yq3t2J.png) 

It would be great to know how to export with this option enabled.

Thanks!

**Comment by dfinke** [comment]
There is a `-Password` param on `Export-Excel` I don't know of a way to set up a read-only mode, there may be a way.
